package.json

```diff
@@ -76,11 +86,6 @@
                 "title": "%containerApps.deployImageApi%",
                 "category": "Azure Container Apps"
             },
-            {
```

Comment:
Why did you move this? I thought you might have moved it next to `createManagedEnvironment`  but it seems somewhat random?

Reply:
I put the `createManagedEnvironment` command out from in between the container app commands and up into the managed environment commands section.  It's just misleading because `createContainerApp` is actually a managed environment command even though it has the word container app in it